### ISSUE TITLE
[RFC] How can I query the API for a list of all scenes?

### ISSUE THREAD

**Issue by SaleenTheGod** [stashapp/stash#2628]
# How can I query the API for a list of all scenes?

*### Scope*
Hello and greetings all. I am trying to figure out how I can query the API and get a list of ALL scenes in my Stash Instance? I am admittedly not strong with graphql but I am pretty familiar with APIs in general. I wasn't able to find out how to do this. I apologize if this has been asked before as well, but I couldn't find anything on it either. I do know there is a **findScene** function, however I wasn't able to determine how to effectively use that, since I believe it wants a scene ID. I just wanna get ALL SCENES in a long JSON format or something. 

## Examples
Long story short, I wanna be able to query Stash for ALL my Scenes and their titles/ids AND/OR query Stash for a title of a scene and have it return back if it finds that scene with all the relevant information. Both would be amazing.

**Comment by stg-annon** [comment]
check out `localhost:9999/playground` It will help you out with learning about the API which is GQL

you also might find looking at the "network" tab in your browsers developer tools to see examples of how the Stash UI interacts with the backed API

what you are looking for specifically is the `findScenes` query an example might be the following

```
query FindScenes($filter: FindFilterType, $scene_filter: SceneFilterType, $scene_ids: [Int!]) {
  findScenes(filter: $filter, scene_filter: $scene_filter, scene_ids: $scene_ids) {
    count
    scenes {  
      id
      checksum
      oshash
      title
      details
    }
  }
}
```
using the variables

```json
{
  "filter": {
	  "direction": "DESC",
	  "page": 1,
	  "per_page": 40,
	  "q": "my search query",
	  "sort": "created_at"
  },
  "scene_filter": {}
}
```
